src/components/QuizGeneratorForm.js

```diff
@@ -0,0 +1,163 @@
+    setNoOfQuestions(e.target.value);
+  };
+  const timerHandler = (e) => {
+    setTimer(e.target.value);
+  };
+  const operationSelectionHandler = (e) => {
+    const operatorIndex = operators.indexOf(e.target.defaultValue);
+    if (operatorIndex >= 0) {
+      updateOprator(operators.filter((operator) => operator !== e.target.defaultValue));
+    } else {
+      updateOprator([...operators, e.target.defaultValue]);
+    }
+  };
+  const onSubmitTapped = () => {
+    if ((minLimit === '') || (minLimit < 1)) {
+      alert('Please provide correct min limit.');
+    } else if ((maxLimit === '') || (maxLimit < 1)) {
+      alert('Please provide correct max limit.');
+    } else if ((noOfQuestions === '') || (noOfQuestions < 1)) {
+      alert('Please provide correct no of questions.');
+    } else if ((timer === '') || (timer < 1)) {
+      alert('Please provide correct timer.');
+    } else if (operators.length === 0) {
+      alert('Please select atleast one operator.');
+    }
+    const quiz = {
+      minLimitVal: minLimit,
+      maxLimitVal: maxLimit,
+      noOfQuestionsVal: noOfQuestions,
+      timerVal: timer,
+      operatorsVal: operators,
+      quizId: currentQuizesCount + 1,
+    };
+    updateQuizs(quiz);
+    setMinLimit('');
+    setMaxLimit('');
+    setNoOfQuestions('');
+    setTimer('');
+    updateOprator([]);
+  };
+  const plusChecked = operators.indexOf('+') >= 0;
+  const minusChecked = operators.indexOf('-') >= 0;
+  const multiplyChecked = operators.indexOf('*') >= 0;
+  const devideChecked = operators.indexOf('/') >= 0;
+  return (
+    <div className="QuizGeneratorForm">
+      <h2>Quiz Generation</h2>
+      <div className="QGFTop">
+        <div>
+          <div className="QZInput">
+            Min Limit
+            <input
+              type="number"
+              className="Input"
+              value={minLimit}
+              onChange={minLimitChangeHandler}
+            />
+          </div>
+          <div className="QZInput">
+            No of Questions
+            <input
+              type="number"
+              className="Input"
+              value={noOfQuestions}
+              onChange={noQuestionHandler}
+            />
+          </div>
+        </div>
+        <div>
+          <div className="QZInput">
+            Max Limit
+            <input
+              type="number"
+              className="Input"
+              value={maxLimit}
+              onChange={maxLimitHandler}
+            />
+          </div>
+          <div className="QZInput">
+            Timer
+            <input
+              type="number"
+              className="Input"
+              value={timer}
+              onChange={timerHandler}
+            />
+          </div>
+        </div>
+      </div>
+      <div className="QGFMid">
+        Operations:
+        <div className="QGFODiv">
+          <div>
+            +
+            <input
+              type="checkbox"
+              id="Operations"
```

Comment:
You've used the same id multiple times. Why specify an ID and then not honor the convention for assigning ids?

Reply:
Done

Reply:
I still see the same id being used multiple times...